microsoft-365/1.1/support/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Supported platforms
+---
+
+The following are the supported platforms for the Alfresco Collaboration Connector for Microsoft 365 version 1.1.3:
+
+| Version | Notes |
+| ------- | ----- |
+| Alfresco Content Services 7.2.0 | |
+| Alfresco Content Services 7.1.0 | |
+| Alfresco Content Services 7.0.0 | |

```

Comment:
According to the [Alfresco Product Support Status page](https://www.alfresco.com/services/subscription/technical-support/product-support-status), MS 365 1.1 is also compatible with ACS 7.3: 

```suggestion
| Alfresco Content Services 7.3.x | |
| Alfresco Content Services 7.2.x | |
| Alfresco Content Services 7.1.x | |
| Alfresco Content Services 7.0.x | |
```